tests/unit_tests/test_wave_06.py

```diff
@@ -143,16 +157,33 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # Assert
+    # result is truthy
+    assert result
+    
+    # tai and jesse's inventories are the correct length
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    
+    # all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
+    assert item_f in tai.inventory
+    assert item_b in tai.inventory
+    assert item_a in tai.inventory
+    
+    assert item_c in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_d in jesse.inventory
```

Comment:
Is there a way to do this with a loop maybe? But if done with a loop what would be the `time complexity of this this functionality?